src/js/classes/Table.js

```diff
@@ -102,28 +102,77 @@ class BarGraphCell extends Cell {
 class DistributionBarCell extends Cell {
   constructor(content, className) {
     super(className);
-    this.content = content["values"];
+    this.values = content["values"].filter(dist => dist["value"] !== 0);
+    this.tooltipName = content["name"];
     this.render();
   }
 
+  createDistributionTable() {
```

Comment:
I'm not sure it makes sense for the tooltips to be tables. Using tables makes sense semantically when we need to do things like sorting, like we are for the larger tables. I think it'd be better to do these tooltips with just regular divs, since that gives us more control over the display.

Reply:
Switched to divs, let me know what you think!